teresa

```diff
@@ -0,0 +1,20 @@
+HTTP/1.1 302 Found

```

Comment:
Oops, I think you committed the output. Also, Github gives a 302 when you try to get the binary, you should follow this binary. 

Here's a one liner:

```
curl -L https://github.com/luizalabs/teresa/releases/download/`curl -s -o- https://github.com/luizalabs/teresa/releases/latest | sed 's/.*tag\/\(.*\)\".*/\1/'`/teresa-linux-amd64 -o /usr/local/bin/teresa && chmod +x /usr/local/bin/teresa
```

With that, you can skip checking for wget as well. 

It is preferable to keep your binary on `/usr/local/bin` instead of `/usr/bin`. It will be already set to PATH.

Reply:
@vinicyusmacedo Damm, I'll remove this file, I didn't notice :(

I did the changes, could you review again pls :)

Reply:
No problems, I'll review it again.